crates/sui-types/src/execution_status.rs

```diff
@@ -237,7 +237,8 @@ pub enum CommandArgumentError {
         "Invalid usage of value. \
         Mutably borrowed values require unique usage. \
         Immutably borrowed values cannot be taken or borrowed mutably. \
-        Taken values cannot be used again."
+        Taken values cannot be used again. \
+        Taken values cannot be implicitly passed by mutable or immutable reference."
```

Comment:
@rockbmb , I'm not sure I follow what you're trying to say here, on top of the existing message -- could you clarify?.  With PTBs, there is only implicit referencing (you cannot take an explicit reference of an input or result)

Reply:
Here's a quick MRE:

```
module move_test::test {
    struct Test { x: u64 }

    fun mut_borrow_on_taken(t: &mut Test): u64 {
        0
    }

    #[test]
    fun test() {
        let test: Test = Test { x: 0 };
        mut_borrow_on_taken(test);
    }
}
```

The Move compiler will correctly error with:
```
Given: '(move_test=0x0)::test::Test'
Expected: '&mut (move_test=0x0)::test::Test'
```

I tried doing this with a PTB (on purpose, just to see what would happen in our particular use-case), and I got `InvalidValueUsage`, so the note I added was meant to cover this particular circumstance.